scripts/haml.rb

```diff
@@ -1,12 +1,13 @@
-require 'rubygems'
-require 'bundler/setup'
-require 'haml'
+# frozen_string_literal: true
+require "rubygems"
+require "bundler/setup"
+require "haml"
 
-require('./styleguide/haml_locals.rb')
+require("./styleguide/haml_locals")
```

Comment:
Minor thing, can we make this `require_relative "../styleguide/haml_locals"` - This assumes something being on the load path, which technically speaking it isn't. The repo is kinda set up like a gem and kinda isn't. So we need to make tackle that later on.

Reply:
Will test that as a follow up change